mist-api/src/middlewares/parse-criteria-for-many.js

```diff
@@ -14,9 +14,13 @@ module.exports = function(app, middlewares) {
 	let criteriaService = app.get('services').criteria,
 		CriteriaError = app.get('errors').CriteriaError
 
-	return (primaryModel, criteriaOptions = {}) => {
+	/**
+	 * {input} is optional string input
+	 */
+	return (primaryModel, criteriaOptions = {}, isPOST = false) => {
```

Comment:
Instead of checking if the request is a POST or not via `isPost`, recommend making this more specific by replacing `isPost` with the actual input object itself:

```js
	return (primaryModel, criteriaOptions = {}, input = null) => {
```

Then the consumer of this method can decide where `input` comes from and this method is not burdened with this logic.